motr/m0crate/crate_index.c

```diff
@@ -382,6 +450,11 @@ static int cr_idx_w_init(struct cr_idx_w *ciw,
 	ciw->nr_kv_per_op = wit->num_kvs;
 	ciw->nr_keys *= ciw->nr_kv_per_op;
 	ciw->ordered_keys = wit->keys_ordered;
+	
+	/** If key size is random, generate it using rand().*/	
+	if (ciw->wit->key_size < 0)  
+		ciw->wit->key_size = sizeof(struct m0_fid) + 
```

Comment:
instead of m0_fid size, can we have ciw->wit->min_key_size

Reply:
Fixed.